requirements/nightly.txt

```diff
@@ -0,0 +1,173 @@
+# SHA1:bc5ff1982b52c217e22f97e94aac08fcfeb13980
+#
+# This file is autogenerated by pip-compile-multi
+# To update, run:
+#
+#    pip-compile-multi
+#
+asttokens==2.4.0
+    # via stack-data
+backcall==0.2.0
+    # via ipython
+certifi==2023.7.22
+    # via requests
+charset-normalizer==3.3.0
+    # via requests
+click==8.1.7
+    # via dask
+cloudpickle==3.0.0
+    # via dask
+comm==0.1.4
+    # via ipywidgets
+confuse==2.0.1
+    # via scipp
+contourpy==1.1.1
+    # via matplotlib
+cycler==0.12.1
+    # via matplotlib
+dask==2023.5.0
+    # via -r nightly.in
+decorator==5.1.1
+    # via ipython
+executing==2.0.0
+    # via stack-data
+fonttools==4.43.1
+    # via matplotlib
+fsspec==2023.9.2
+    # via dask
+graphlib-backport==1.0.3
+    # via
+    #   sciline
+    #   scipp
+graphviz==0.20.1
+    # via -r nightly.in
+h5py==3.10.0
+    # via
+    #   scippneutron
+    #   scippnexus
+idna==3.4
+    # via requests
+importlib-metadata==6.8.0
+    # via dask
+importlib-resources==6.1.0
+    # via matplotlib
+ipydatawidgets==4.3.5
+    # via pythreejs
+ipython==8.12.3
+    # via ipywidgets
+ipywidgets==8.1.1
+    # via
+    #   ipydatawidgets
+    #   pythreejs
+jedi==0.19.1
+    # via ipython
+jupyterlab-widgets==3.0.9
+    # via ipywidgets
+kiwisolver==1.4.5
+    # via matplotlib
+locket==1.0.0
+    # via partd
+matplotlib==3.7.3
+    # via plopp
+matplotlib-inline==0.1.6
+    # via ipython
+numpy==1.24.4
+    # via
+    #   contourpy
+    #   h5py
+    #   ipydatawidgets
+    #   matplotlib
+    #   pythreejs
+    #   scipp
+    #   scippneutron
+    #   scipy
+packaging==23.2
+    # via
+    #   dask
+    #   matplotlib
+    #   pooch
+parso==0.8.3
+    # via jedi
+partd==1.4.1
+    # via dask
+pexpect==4.8.0
+    # via ipython
+pickleshare==0.7.5
+    # via ipython
+pillow==10.1.0
+    # via matplotlib
+platformdirs==3.11.0
+    # via pooch
+plopp @ git+https://github.com/scipp/plopp@main
+    # via -r nightly.in
+pooch==1.7.0
+    # via scippneutron
+prompt-toolkit==3.0.39
+    # via ipython
+ptyprocess==0.7.0
+    # via pexpect
+pure-eval==0.2.2
+    # via stack-data
+pygments==2.16.1
+    # via ipython
+pyparsing==3.1.1
+    # via matplotlib
+python-dateutil==2.8.2
+    # via
+    #   matplotlib
+    #   scippnexus
+pythreejs==2.4.2
+    # via -r nightly.in
+pyyaml==6.0.1
+    # via
+    #   confuse
+    #   dask
+requests==2.31.0
+    # via pooch
+sciline @ git+https://github.com/scipp/sciline@main
+    # via -r nightly.in
+scipp==23.8.0
+    # via
+    #   -r nightly.in
+    #   scippneutron
+    #   scippnexus
+scippneutron @ git+https://github.com/scipp/scippneutron@main
+    # via -r nightly.in
+scippnexus==23.8.0
```

Comment:
Do we need to also get `scippnexus` from `git`?

Reply:
Hmm, good point, we need to allow for listing extra deps that are transitive.

Reply:
See update in scipp/copier_template#43